index.js

```diff
@@ -359,8 +360,14 @@ DumbPipe.registerOpener("camera", function(message, sender) {
     audio: false,
   }, function(localMediaStream) {
     mediaStream = localMediaStream;
+    url = URL.createObjectURL(localMediaStream);
 
-    video.src = URL.createObjectURL(localMediaStream);
+    video.onerror = video.onloadeddata = () => {
+      URL.revokeObjectURL(url);
+      video.removeAttribute('src');
```

Comment:
It is not required to remove the `src` attribute. The object url is a plain string. Holding it won't increase the references to the target object.